.travis.yml

```
@@ -8,6 +8,9 @@ language: php
 php:
     - '7.1'
     - '7.2'
+    - '7.3'
+    - '7.4'
+    - '8.0'
```

I think you need to use snapshot here as Travis does not have `8.0` image yet.
```suggestion
    - '8.0snapshot'
```